nullshinsa-product/src/main/java/com/example/nullshinsaproduct/product/domain/entity/Product.java

```diff
@@ -0,0 +1,46 @@
+package com.example.nullshinsaproduct.product.domain.entity;
+
+import com.example.nullshinsaproduct.common.domain.BaseEntity;
+import com.example.nullshinsaproduct.product.domain.entity.Seller;
+import com.example.nullshinsaproduct.product.domain.enumeration.CouponApplyPossible;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Product extends BaseEntity {
```

Comment:
@AllArgsConstructor

Reply:
order 
order_item

OrderDAO {
  OrderVO getOrder();
  List<OrderItem> getOrderItem();
  OrderInfoVO getOrderInfo(){
    
    return new  ( this.getOrder(), this.getOrderItem());   
}
}